json/cards/Aquapolis.json

```
@@ -8111,7 +8111,7 @@
     "id": "ecard2-141",
     "name": "Weakness Guard",
     "imageUrl": "https://images.pokemontcg.io/ecard2/141.png",
-    "subtype": "",
+    "subtype": "Pokémon Tool",
```

Weakness Guard is not a Tool, seems to be similar to PlusPower and Defender.

```suggestion
    "subtype": "",
```